### PR TITLE
fix(driver proxy): exceptions don't have enough information

### PR DESCRIPTION
**Pull request by stakach**
to be able to locate where the request came from

### CODE REVIEW

File: src/placeos-driver/proxy/driver.cr [hunk 1]
```diff
@@ -104,8 +104,14 @@ class PlaceOS::Driver::Proxy::Driver
...
-          @system.logger.warn(exception: exception) { exception.message }
-          raise exception
+          runtime_error = RuntimeError.new "remote exception", exception
+
+          begin
+            raise runtime_error
+          rescue error_with_backtrace
+            @system.logger.warn(exception: error_with_backtrace) { exception.message }
+            raise error_with_backtrace
+          end
```
Comment (caspiano) [inline src/placeos-driver/proxy/driver.cr:114]: Any reason you can't do this?
```suggestion
          runtime_error = RuntimeError.new "remote exception", exception
          @system.logger.warn(exception: error_with_backtrace) { exception.message }
          raise runtime_error
```

Comment (caspiano) [reply src/placeos-driver/proxy/driver.cr:114]: Isn't that what's essentially happening already?

Comment (stakach) [reply src/placeos-driver/proxy/driver.cr:114]: as there is no backtrace unless you raise the error

Comment (caspiano) [inline src/placeos-driver/proxy/driver.cr:114]:
```suggestion
          begin
            raise PlaceOS::Driver::RemoteException.new(result.payload, error, backtrace)
          rescue exception
            @system.logger.warn(exception: exception) { exception.message }
            raise exception
          end
```

**Review by caspiano — APPROVED** [review]

**Pull request merged**